.travis.yml

```diff
@@ -41,13 +34,17 @@ env:
     - secure: "iVVToSZWmR+9Y8mXVKiWl2p0mfS9hAQxW5dAeNlXQk9ara/2cvmdffOmLLGkhI5caZvQSVjne5enkng1Y6Ifns+heuhqiPS8495eds+PKl/70YNUgoIOXUWEehimHnNLg6R9q7uxrqy8rql89LAXZaeJImVnRrm0LoZ9jgVYivCqUpUcUxJ3CsFkZwJv5LSAFI964DaG2rsj2fiNixx6tnUAssQMGpoWoo9o+psb4Uwyr8S1rABCiMt494iw7RenlhCIFv2O04+p305Nx8wy11n9tV1ViK7tKF819Lw/iXB+5DA4CrT+ff56EdPWJ4wik7CEtydKcjAx0C00s04H93GuGoyWqFLFNliNj54d8BtRu0YZBhl9m51N3dPxq+usTrdV/veL0IJ7pNqK3CV+IKc9Qb+xHNuuYgcZaMMKKaB4+5TQR/pCe+i9v3shJW+zXUWsv0WS2zTIabZNWHNlTb9aPTyHPSyf9CbB04qFa4mcza/ohhhA6pCin9ZScHCtb45DHPMe6/khQIztkuyTwBWpLWYEjnbeOjOFgRKf1Zt2AYi8yR6Vcv9gDxrYu+KOqIuqKpS7Yn1Fcktsn7ty/rTr6XWlmhg3TLKRG6C2kTNNwxJmBo4WPcsENeuS5JQjsXp46ZWFeC4VsLZe7p62bM4QjtECqWX1I5rtiCShKIc="
     # REVIEWDOG_GITHUB_API_TOKEN
     - secure: "eVA62wnIJW4/hZ1OCDQCAw+MvbvNy18yN5LkVtFz/ZEB4Y8k5BJz1RNWPrO7ymOUA3O4wcqVjiMlEd6JgTvsuKX/Q5acLw66RO0NUEovRaC7hunTMBxby/kQs8jR7238yC2GAhTSN0bIqnhwejSb+ERN/J7hx8dack6VJppoJ4sIkYtMzTZOPcVfvdxAa5NCzZCIqZTJKOfXX3srFhNaadCq9FQLDGHyehMqgA0eDUf+WuwXPJ/a4oWTkucHl8XQTxSb0cqxSrAX8MHpINU/UcHu7l9/lE2MWS08bQVS+FPp58PldsNtjNDFN7f/0g30XYs+xCNX9fGxRlTxPD5pvB32WeWT+xxzQCdkku9QGch7/RY4g8+5rEIfsm/aMVKai9CaLJJJDTMnXBfhSSPXNRWfbDg7pOae0kdz4+Y+Q2hHcQHDdBBzQBZ7458ujImsXbqS3jhWE5Ogh88jTw9iUiYdPDTFO0TR5JLXG6k6x7OrmurDe5SxH29NPHJl+N8KjH9XHWTVj4mKJPOLDLzwPzBvZsEr8DfB07+2PtooHMLedy9jE65sOOkwVywO7bPFtk2GtpHFYLOzCtnN9aGw3OWni46Q1vVYIf47k6bO2rn2yK/rCTRUEGTCcddWcbUQFtteFe8B5cajShdZIXoQ3C6bl+JbuDzOSHHGECuBgaI="
+    - GO_VERSION=1.14.x
 
   matrix:
-    # what we used for the latest PMM release
-    - GO_VERSION=1.14.x
+    # There is no case when pmm-update is older than pmm-managed (pmm-update self-updates first),
+    # so there is no need to test with old PMM Server versions.
+    # However, there may be a need to use a different devcontainer image once
+    # https://jira.percona.com/browse/PMM-5451 is implemented.
+    - PMM_SERVER_IMAGE=perconalab/pmm-server:dev-latest
 
-    # the latest HEAD
-    - GO_VERSION=tip
+    # To test with some feature build
+    # - PMM_SERVER_IMAGE=perconalab/pmm-server-fb:PR-123-4567890
```

Comment:
While seeing this line is commented, just want to warn that I removed `- GO_VERSION=tip` with the intention to revert whole  `.travis.yml`

Reply:
That should be a reason for rejecting PR